src/main/resources/blockchain/default-genesis.json

```diff
@@ -1,5 +1,5 @@
 {
-  "difficulty": "0x0400000000",
+  "difficulty": "0x040000",
```

Comment:
no it was not intentional there should be `0x0400000000` this is the value from main chain